fern/docs.yml

```diff
@@ -467,35 +467,35 @@ navigation:
             path: ./pages/fern-docs/content/reusable-snippets.mdx
           - page: Changelog
             path: ./pages/docs/writing-content/changelog.mdx
-            icon: fa-regular fa-clock
+            icon: fa-regular fa-clock-rotate-left
 
       - section: API References
         contents:
           - page: Generate API Reference
-            icon: fa-regular fa-gear-code
+            icon: fa-regular fa-gear
```

Comment:
we use gear already, thoughts on `square-plus`? or `folder-plus`?

Reply:
I want to emphasize that it's automatically generated. going to try `fa-solid fa-wand-magic-sparkles`